### ISSUE TITLE
Accessibility: Insufficient contrast on 'Find out more' link on Sponsorship Page

### ISSUE THREAD

**Issue by robcrowley** [dddwa/dddperth-website#55]
The ratio is 1.8:1 but will need to be at least 4.5:1 for users with low vision to be able to read it.

**Recommendation:** choose a darker `background-color` such as the existing logo green.

**Comment by robdmoore** [comment]
Is this now fixed? cc @Macca2805, @robcrowley, @matthewjward

**Comment by Macca2805** [comment]
Yes, the button text contrast should be more appropriate now. We didn't need to change the button class.
